packages/database/src/migrations/20210610060144-AddFluTrackerLGAPercentNonFirstNationsILI-modifies-data.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+import { deleteObject, generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const INDICATOR = {
+  id: generateId(),
+  code: 'FluTracker_LGA_Percent_Non_First_Nations_ILI',
+  builder: 'analyticArithmetic',
+  config: {
+    formula:
+      'FWV_LGA_003b == -1 or FWV_LGA_003b == -1 or FWV_LGA_003b == 0 ? undefined : FWV_LGA_004b / FWV_LGA_003b',
```

Comment:
Is `undefined` supported here? I didn't realised mathjs native supported it!

Reply:
Emm it did support, but Chris suggested me to use 'undefined' and I forgot to change this one. To make them consistent I will add quotes to them.